common/src/View/MapDocument.cpp

```diff
@@ -1188,12 +1205,21 @@ namespace TrenchBroom {
             }
 
             for (auto* minuend : minuends) {
-                const std::vector<Model::Brush*> result = minuend->subtract(*m_world, m_worldBounds, currentTextureName(), subtrahends);
-
-                if (!result.empty()) {
-                    kdl::vec_append(toAdd[minuend->parent()], result);
+                auto result = minuend->subtract(*m_world, m_worldBounds, currentTextureName(), subtrahends);
+                if (result.is_success()) {
+                    const auto brushes = kdl::get_value(std::move(result));
```

Comment:
Too many autos in this code block, maybe mark this one as `const std::vector<Model::Brush*> brushes = `?